packages/components/src/components/form/Checkbox/Checkbox.tsx

```diff
@@ -87,13 +89,19 @@ export const Checkbox = ({
 
     return (
         <Wrapper
+            data-test={dataTest}
             isDisabled={isDisabled}
             onClick={handleClick}
             onKeyUp={handleKeyUp}
             tabIndex={0}
             {...rest}
         >
-            <IconWrapper $color={color} isChecked={isChecked} isDisabled={isDisabled}>
+            <IconWrapper
+                data-test="@checkbox/icon-wrapper"
```

Comment:
Why is it needed here?

Reply:
Removed, 9da580e1edb8550f24d0d4bcedf138449ef1f283 
It was a leftover from something I was experimenting.